python/smashbox/utilities/__init__.py

```diff
@@ -26,11 +24,26 @@ def setup_test():
     If exception is raised then the testcase execution is aborted and smashbox terminates with non-zero exit code,
 
     """
-    reset_owncloud_account(num_test_users=config.oc_number_test_users)
-    reset_rundir()
-    reset_server_log_file()
```

Comment:
These functions are owncloud specific, so I moved them to owncloud sync client class.